cfnet/datasets.py

```diff
@@ -21,35 +23,32 @@ def __len__(self):
     def __getitem__(self, idx):
         return self.X[idx], self.y[idx]
 
-
 # %% ../nbs/01_datasets.ipynb 5
 # copy from https://jax.readthedocs.io/en/latest/notebooks/Neural_Network_and_Data_Loading.html#data-loading-with-pytorch
 def _numpy_collate(batch):
-    if isinstance(batch[0], np.ndarray):
-        return np.stack(batch)
-    elif isinstance(batch[0], (tuple, list)):
-        transposed = zip(*batch)
-        return [_numpy_collate(samples) for samples in transposed]
-    else:
-        return np.array(batch)
-
+  if isinstance(batch[0], np.ndarray):
+    return np.stack(batch)
+  elif isinstance(batch[0], (tuple,list)):
+    transposed = zip(*batch)
+    return [_numpy_collate(samples) for samples in transposed]
+  else:
+    return np.array(batch)
 
 class DataLoader:
-    def __init__(
-        self,
-        dataset: Dataset,  # dataset, a Dataset object
-        batch_size: int = 1,  # batch size
-        shuffle: bool = False,  # if true, dataloader shuffles before sampling each batch
-        seed: int = 42,  # seed for random number generator
-        sampler=None,
-        batch_sampler=None,
-        num_workers=0,
-        collate_fn=_numpy_collate,  # collate function, default is _numpy_collate()
-        pin_memory=False,
-        drop_last: bool = False,  # if true, dataloader drops the last batch that is less than the batch size
-        timeout=0,
-        worker_init_fn=None,
-    ):
+    def __init__(self,
```

Comment:
Use original format.

We want to follow the [black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) style.